### ISSUE TITLE
Publish API for single messages

### ISSUE THREAD

**Issue by patriknw** [akka/alpakka-kafka#500]
Sometimes you don't need a stream, but just want to publish to Kafka. We could provide something like this:

```
import scala.concurrent.Future
import scala.concurrent.Promise

import akka.Done
import akka.kafka.ProducerMessage.Message
import org.apache.kafka.clients.producer.Callback
import org.apache.kafka.clients.producer.Producer
import org.apache.kafka.clients.producer.RecordMetadata

class PublishOneExample[K, V, P](producer: Producer[K, V]) {

  def publish(msg: Message[K, V, P]): Future[Done] = {
    val result = Promise[Done]
    producer.send(msg.record, new Callback {
      override def onCompletion(metadata: RecordMetadata, exception: Exception): Unit = {
        if (exception == null)
          result.success(Done)
        else
          result.failure(exception)
      }
    })
    result.future
  }

}
```

**Comment by Reeebuuk** [comment]
And vice versa, we want to consume only one message and if message isn't there in certain timeout end the stream. 

To specify from which offset to start from and how many messages to read. 

I that's possible now please shout :)

**Comment by WayneWang12** [comment]
Any update for this issue?

**Comment by ennru** [comment]
Implemented with #1085
